### ISSUE TITLE
Error when loading at server 1.16.5 Spigot start: java.lang.NoClassDefFoundError: kotlin/collections/CollectionsKt

### ISSUE THREAD

**Issue by serious-angel** [Sytm/waypoints2#57]
**Describe the bug**
Dear Developers,

Thank you very much for the project.

I'm sorry, but for some reason the plugin (`v3.2.0`) doesn't load on `Spigot-1.16.5`. The log part:

```
[01:39:09] [Server thread/INFO]: [Waypoints] Enabling Waypoints v3.2.0                                                                                                                                                                 [32/1849]
[01:39:09] [Server thread/ERROR]: Error occurred while enabling Waypoints v3.2.0 (Is it up to date?)
java.lang.NoClassDefFoundError: kotlin/collections/CollectionsKt
        at de.md5lukas.waypoints.config.general.CustomIconFilterConfiguration.<init>(CustomIconFilterConfiguration.kt:16) ~[?:?]
        at de.md5lukas.waypoints.config.general.GeneralConfiguration.<init>(GeneralConfiguration.kt:20) ~[?:?]
        at de.md5lukas.waypoints.config.WaypointsConfiguration.<init>(WaypointsConfiguration.kt:12) ~[?:?]
        at de.md5lukas.waypoints.WaypointsPlugin.loadConfiguration(WaypointsPlugin.kt:86) ~[?:?]
        at de.md5lukas.waypoints.WaypointsPlugin.onEnable(WaypointsPlugin.kt:65) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:351) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:494) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:408) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:435) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:218) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:809) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:164) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at java.lang.Thread.run(Thread.java:832) [?:?]
Caused by: java.lang.ClassNotFoundException: kotlin.collections.CollectionsKt
        at java.net.URLClassLoader.findClass(URLClassLoader.java:435) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:167) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:96) ~[spigot-1.16.5.jar:2991-Spigot-018b9a0-f3f3094]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
        ... 15 more
```

Just to clarify, is the issue local? If so, is it known how to resolve it?

Best and kind regards

**To Reproduce**
1. Download the plugin;
2. Put to `.../spigot-1.16.5_server/plugins/`;
3. Start the server;
4. The error message appears in the server console log.

**Expected behavior**
The plugin loads.

- [x] I am using the latest version of Waypoints available

**Comment by Sytm** [comment]
> Minimum Spigot version is 1.16.5 (recent builds)

You need to build 1.16.5 again with BuildTools. The required functionality has been added in later builds.
The version of Spigot you are currently running is outdated for almost 100 builds

**Comment by serious-angel** [comment]
Built with `BuildTools.jar`, and indeed a different error appeared. It seems it's related to Dynmap (`Dynmap version: core=3.3-beta-1-564, plugin=3.3-beta-1-564`).
```
[07:39:18] [Server thread/ERROR]: Error occurred while enabling Waypoints v3.2.0 (Is it up to date?)
java.lang.NullPointerException: markerApi.getMarkerIcon(…integrations.dynmap.icon) must not be null
        at de.md5lukas.waypoints.integrations.DynMapIntegration.setupDynMap(DynMapIntegration.kt:37) ~[?:?]
        at de.md5lukas.waypoints.WaypointsPlugin.initIntegrations(WaypointsPlugin.kt:154) ~[?:?]
        at de.md5lukas.waypoints.WaypointsPlugin.onEnable(WaypointsPlugin.kt:74) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:480) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugin(CraftServer.java:492) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.enablePlugins(CraftServer.java:406) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at net.minecraft.server.v1_16_R3.MinecraftServer.loadWorld(MinecraftServer.java:554) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:257) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:928) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273) ~[spigot-1.16.5.jar:3096a-Spigot-9fb885e-af1a232]
        at java.lang.Thread.run(Thread.java:832) [?:?]
```

**Comment by Sytm** [comment]
That's related to #58 and has been fixed in version 3.2.1
